scripts/generateTypeScriptTypesFromSchema.ts

```diff
@@ -15,7 +15,8 @@ async function generate(): Promise<void> {
   try {
     await generateTypeScriptTypes(schema, typeDefinitionFileName, {
       customScalarType: {
-        'JSON': 'JSON'
+        JSON: 'JSON',
```

Comment:
Denne brukes kun for customFields så om den ikkje lenger trengs kan den fjernes.